src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,46 @@
+package parser;
+
+import com.google.gson.Gson;
+import org.junit.jupiter.api.Test;
+import shop.Cart;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+    private Cart cart = new Cart("test-andrew-cart");
+    private Gson gson = new Gson();
+
+    @Test
+    void readFromFile() throws IOException {
```

Comment:
Could you please explain what do you test? As of now I see that you are create file with no data in it. Also why you are using the same name of the file that already created ?